Dockerfile

```diff
@@ -30,4 +27,11 @@ RUN apt-get update -y && \
     curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
     apt-get install -y nodejs && npm install
 
+# Install prettier-daemon
+COPY prettierd-server.sh /usr/local/bin/prettierd-server
+
+# Install custom entrypoint
+COPY entrypoint.sh /usr/local/bin/cf-entrypoint
+CMD ["cf-entrypoint"]
+
```

Comment:
We need those new lines in .docker/web/Dockerfile too

Reply:
@jr-cologne can you take a look at this? I'm not even sure why we use two Dockerfiles 😃